server/src/test/java/org/elasticsearch/search/query/SearchQueryIT.java

```diff
@@ -1758,4 +1761,19 @@ public void testFieldAliasesForMetaFields() throws Exception {
        }
 
    }
+
+    /**
+     * Test fix for NPE from {@link SpanBooleanQueryRewriteWithMaxClause#rewrite(IndexReader, MultiTermQuery)}.
+     * See https://github.com/elastic/elasticsearch/issues/52894 for details
+     */
+    public void testIssue52894() {
```

Comment:
Can you change to a more descriptive name ? I don't think we should add a link to an issue in the comment either. This test checks that `span_fuzzy` query are correctly handled and #52894 revealed that it was missing.